.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+  - '3.6'
+install:
+  - 'pip install pipenv'
+  - 'pip install coveralls'
```

Comment:
since we have coveralls and coverage in our pipfile we do not need to instruct travis to install them again